initfiles/componentfiles/thor/run_thor

```diff
@@ -54,6 +54,14 @@ while [ 1 ]; do
     else
         nslaves=`cat $instancedir/uslaves.start | wc -l`
         $deploydir/frunssh $instancedir/uslaves.start "/bin/sh -c '$deploydir/start_slaves %a $THORMASTER $THORMASTERPORT $logdir $instancedir $deploydir $THORNAME $PATH_PRE $logredirect'" -i:$SSHidentityfile -u:$SSHusername -pe:$SSHpassword -t:$SSHtimeout -a:$SSHretries -n:$nslaves 2>&1
+        FRUNSSH_RC=$?
+        if [[ ${FRUNSSH_RC} -gt 0 ]]; then
+          echo "frunssh could not connect to all slave machines"
+          echo "check your key and ssh settings across the cluster"
```

Comment:
Does the error code necessarily mean 'could not connect' ? 
Wouldn't frunssh now return the same code if, e.g. the script it was executing failed?